code/modules/surgery/organs/autoimplanter.dm

```diff
@@ -6,41 +6,39 @@
 	icon_state = "autoimplanter"
 	item_state = "walkietalkie"//left as this so as to intentionally not have inhands
 	w_class = 2
-	var/obj/item/organ/cyberimp/storedorgan
+	var/obj/item/organ/storedorgan
+	var/organ_type = /obj/item/organ/cyberimp
 	var/uses = INFINITE
-	var/one_use = 0//separate var, as a 2-use device can have 1 use remaining if used once but not be a single-use device
 
 /obj/item/device/autoimplanter/New()
 	..()
-	if(one_use)
-		uses = 1
 	if(storedorgan)
 		storedorgan.loc = src
 
 /obj/item/device/autoimplanter/attack_self(mob/user)//when the object it used...
 	if(!uses)
 		user << "<span class='warning'>[src] has already been used. The tools are dull and won't reactivate.</span>"
 		return
-	if(!storedorgan)
+	else if(!storedorgan)
 		user << "<span class='notice'>[src] currently has no implant stored.</span>"
 		return
 	storedorgan.Insert(user)//insert stored organ into the user
 	user.visible_message("<span class='notice'>[user] presses a button on [src], and you hear a short mechanical noise.</span>", "<span class='notice'>You feel a sharp sting as [src] plunges into your body.</span>")
 	playsound(get_turf(user), 'sound/weapons/circsawhit.ogg', 50, 1)
 	storedorgan = null
 	if(uses != INFINITE)
-		uses --
+		uses--
 	if(!uses)
 		desc = "[initial(desc)] Looks like it's been used up."
 
 /obj/item/device/autoimplanter/attackby(obj/item/I, mob/user, params)
-	if(istype(I, /obj/item/organ/cyberimp))
-		if(one_use && !storedorgan)
-			user << "<span class='notice'>[src] doesn't accept implants. It must be a single-use device.</span>"
-			return//prevents removing the intended implant to insert your own
```

Comment:
Yeah, just a file that I forgot to exclude from this PR. Good thing there was nothing serious, and mechanics of this implanter remain unchanged. 

Modifications are made for organ implanter, and it is not included.